external-crates/move/crates/move-analyzer/editors/code/README.md

```diff
@@ -1,7 +1,9 @@
 # Move
 
 Provides language support for the Move programming language. For information about Move visit the
-language [documentation](https://docs.sui.io/concepts/sui-move-concepts).
+language [documentation](https://docs.sui.io/concepts/sui-move-concepts). It also provides early-stage
```

Comment:
But this isn't actually true, right? It lets you run the tracing command, it requires another extension to do trace debugging.

Reply:
Okay, could we say "The dependent extension, `${fully qualified extension name}`, also provides..." just to be explicit?

Reply:
We do say that later in the README where we describe how to actually trace-debug Move code:

"_If the opened Move source file is located within a buildable project, and you have the `sui`
  binary installed, you can trace-debug execution of Move unit tests within this project.
  This functionality is provided by this (Move) extension automatically including the Move Trace Debugging
  [extension](https://marketplace.visualstudio.com/items?itemName=mysten.move-trace-debuggin)_"

Not sure why we should throw this particular piece of information at the user right at the start, particularly that we don't at all mention that syntax highlighting is provided the same way (through a separate extension), unless you think we should do it as well. That being said, if you think it's crucial, I am not that strongly opposed to adding this info at the beginning, but still wanted to cast my vote here

Reply:
I am going to land it as is but happy to change it (or stamp a change) if need be